packages/integrations/src/aws/src/index.ts

```
@@ -0,0 +1,65 @@
+
+    // 2. Define filters for the findings we want to retrieve.
+    // Example: get only NEW (unresolved) findings for failed compliance controls.
+    const params: GetFindingsCommandInput = {
+      Filters: {
+        WorkflowStatus: [{ Value: "NEW", Comparison: "EQUALS" }],       // only active findings
+        ComplianceStatus: [{ Value: "FAILED", Comparison: "EQUALS" }]  // only failed control checks
+      },
+      MaxResults: 100  // adjust page size as needed (max 100)
+    };
+
+    let command = new GetFindingsCommand(params);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'command' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)